docs/user-guides/dashboard-as-code.md

```diff
@@ -66,6 +66,108 @@ If the build is successful, the result can be found in the generated `built` fol
 > [!NOTE]
 > the `-o` (alternatively '--output') flag is optional (the default output format is YAML).
 
+## Getting started with Go SDK
+
+### Prerequisites
+
+- `percli`, the [CLI of Perses](../tooling/cli.md).
+- `go`, the [programming language](https://go.dev/).
+
+### Repository setup
+
+Create a new folder that will become your DaC repository, then follow the steps below:
+
+#### 1. Initialize the Go module
+
+```
+go mod init <module name>
+```
+See the [Go documentation](https://go.dev/doc/tutorial/create-module) for more information about this step.
+
+#### 2.Install the Perses SDK
+
+```
+go get github.com/perses/perses
+```
+
+If you need a specific version, you can specify it as follows:
+
+```
+go get github.com/perses/perses v0.43.0
+```
+
+## Develop dashboards
+
+You are now fully ready to start developing dashboards as code!
+
+It's first strongly recommended to ramp up on Go if you are not familiar with this technology. For this have a look at:
+- The [official website](https://go.dev/) of Go.
+
+Then, you can check an example of DaC usage [here](../../internal/cli/cmd/dac/build/testdata_go/main.go). This example is heavily relying on the SDK utilities we provide. 
```

Comment:
```suggestion
Then, you can check an example of DaC usage [here](../../internal/cli/cmd/dac/build/testdata_go/main.go).
```

-> the second sentence makes sense in the case of CUE because you are not obliged to use the DaC "builder" utilities provided, but it's not the case here.

Reply:
You can still create dashboards / resources using only Go constructor
```
v1.Dashboard{...}
```

Reply:
Ah indeed, nvm then